pyasdf/asdf_data_set.py

```diff
@@ -1353,8 +1353,8 @@ def ifilter(self, *query_objects):
 
         raise StopIteration
 
-    def process_two_files_without_parallel_output(
-            self, other_ds, process_function, traceback_limit=3):
+    def process_two_files(self, other_ds, process_function,
+                          output_filename=None, traceback_limit=3):
```

Comment:
I add an argument here called "output_filename". If specified, then the output file will be generated